docs/installation/presto-configuration.rst

```diff
@@ -9,74 +10,92 @@ you still may need to make some changes.
-to use a lower value for ``query.max-memory-per-node``.
+   We recommend setting ``query.max-memory-per-node`` to half of the JVM config max memory, though if your workload is highly concurrent, you may want
+   to use a lower value for ``query.max-memory-per-node``. By default, ``query.max-memory-per-node`` is 10% of the
+   ``Xmx`` value specified in ``jvm.config``.
 
 3. Run the following command to deploy the configuration change to the cluster: ::
 
-    sudo ./presto-admin configuration deploy
+        sudo ./presto-admin configuration deploy
 
 
 4. Restart the Presto servers so that the changes get picked up: ::
 
-    sudo ./presto-admin server restart
+        sudo ./presto-admin server restart
 
 
-If you are running Presto in a test environment that has less than 16 GB of memory available,
-you will need to follow similar procedures to set the memory configurations lower.
+   If you are running Presto in a test environment that has less than 16 GB of memory available,
+   you will need to follow similar procedures to set the memory configurations lower.
 
 Log file location configurations
----------------------------------
+--------------------------------
 
 For most production environments, it will be necessary to change the log locations. In order to update these:
 
-1. In ``/etc/opt/prestoadmin/coordinator/node.properties`` and
-``/etc/opt/prestoadmin/workers/node.properties`` specify ``node.data-dir``. Presto stores certain logs and other data in
-this location. It is very important that this location has enough space for the logs on the filesystem on
-each node where Presto is running. The default location for this is ``/var/lib/presto/data``. You can can specify a new
-location in ``node.properties``: ::
+1. Stop the Presto server. ::
 
-    node.data-dir=/disk1/presto/data
+    sudo ./presto-admin server stop
 
-2. There are two more log related properties that should be specified with locations that have enough space, these
-are ``node.launcher-log-file`` and ``node.server-log-file``. You can specify them in ``node.properties``: ::
+2. Presto stores logs and other data in ``node.data-dir``, ``node.launcher-log-file``,
+   and ``node.server-log-file``. It is very important that these locations have enough space for the logs on the filesystem on
+   each node where Presto is running. The default location for ``node.data-dir`` is ``/var/lib/presto/data``, the
+   default location for ``node.launcher-log-file`` is ``/var/log/presto/launcher.log``, and the default
+   location for ``node.server-log-file`` is ``/var/log/presto/server.log``.
+   If these locations have limited space, specify a different location for ``node.data-dir``,
+   ``node.launcher-log-file``, and ``node.server-log-file`` in ``/etc/opt/prestoadmin/coordinator/node.properties`` and
+   ``/etc/opt/prestoadmin/workers/node.properties``: ::
```

Comment:
Maybe add something like:
_Assuming chosen locations are: `/data1` and `/data2` for data directory and server logs respectively, configuration should look as follows:_ ?